js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -497,7 +497,7 @@ const AssetGraphExplorerWithData = ({
     </ShortcutHandler>
   );
 
-  const onFilterToGroup = (group: GroupLayout) => {
+  const onFilterToGroup = (group: AssetGroup) => {
```

Comment:
Was this change necessary? If it is then this should be `AssetGroup | GroupLayout`

Reply:
Yup it was necessary since we don't have access to the `GroupLayout` from the sidebar component